cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala

```diff
@@ -84,88 +84,96 @@ class AggregatingFunctionsTest extends DocumentingTest {
         query("MATCH (n {name: 'A'})-->(x) RETURN count(x)", ResultAssertions((r) => {
           r.toList should equal(List(Map("count(x)" -> 3L)))
         })) {
           p("The number of nodes connected to the start node is returned.")
           resultTable()
         }
       }
-      section("Count non-`null` values") {
-        p("The non-`null` values can be counted by using `count(<expression>)`.")
+      section("Counting non-`null` values") {
+        p("`count(expression)` can be used to return the number of non-`null` values returned by the expression.")
         query("MATCH (n:Person) RETURN count(n.age)", ResultAssertions((r) => {
           r.toList should equal(List(Map("count(n.age)" -> 3L)))
         })) {
-          p("The count of related nodes with the `age` property set is returned.")
+          p("The number of `:Person` nodes having an `age` property is returned.")
           resultTable()
         }
       }
       section("Counting with and without duplicates") {
         p(
-          """In this example we are trying to find all our friends of friends, and count them.
-            |The first aggregate function, `count(DISTINCT friend_of_friend)`, will only see a `friend_of_friend` once -- `DISTINCT` removes the duplicates.
-            |The latter aggregate function, `count(friend_of_friend)`, might very well see the same `friend_of_friend` multiple times.
-            |In this case, both `B` and `C` know `D` and thus `D` will get counted twice, when not using `DISTINCT`.""")
+          """In this example we are trying to find all our friends of friends, and count them:
+            |
+            |* The first aggregate function, `count(DISTINCT friend_of_friend)`, will only count a `friend_of_friend` once, as `DISTINCT` removes the duplicates.
+            |* The second aggregate function, `count(friend_of_friend)`, will consider the same `friend_of_friend` multiple times.""")
         query(
           """MATCH (me:Person)-->(friend:Person)-->(friend_of_friend:Person)
             |WHERE me.name = 'A'
             |RETURN count(DISTINCT friend_of_friend), count(friend_of_friend)""".stripMargin, ResultAssertions((r) => {
             r.toList should equal(List(Map("count(DISTINCT friend_of_friend)" -> 1L, "count(friend_of_friend)" -> 2L)))
           })) {
+          p("Both `B` and `C` know `D` and thus `D` will get counted twice when not using `DISTINCT`.")
           resultTable()
         }
       }
     }
     section("max()", "functions-max") {
-      p("`max()` takes a numeric property as input, and returns the highest value in that column.")
-      function("max(expression)", ("expression", "A numeric expression."))
+      p("`max()` returns the maximum value in a set of values.")
+      function("max(expression)", "Either an Integer or a Float or a String, depending on the values returned by `expression`.", ("expression", "An expression returning a set of numeric or string values."))
```

Comment:
Again, this sounds wrong. It sounds like I could call `max([1,2,3])`, because the expression `[1,2,3]` does indeed _return  a set of numeric or string values_. But that is not how aggregating functions work. You should call `max(n.age)`, where the expression acts on a column and aggregates over all rows.